xml/deployment_cifs.xml

```diff
@@ -328,19 +349,33 @@
       </step>
       <step>
        <para>
-        Initialize the cluster on &si.earth;:
+        Map the RBD image <literal>rbd</literal> on both &sgw;s via
+        <systemitem class="daemon">rbdmap.service</systemitem>.
        </para>
+       <para>
+        Edit <filename>/etc/ceph/rbdmap</filename> and add an entry for the SBD image:
+       </para>
+<screen>rbd/sbd01 id=samba.gw,keyring=/etc/ceph/ceph.client.samba.gw.keyring</screen>
+      <para>
+       Enable and start <systemitem class="daemon">rbdmap.service</systemitem>:
+      </para>
 <screen>
-&prompt.earth;<command>ha-cluster-init</command>
-      </screen>
+&prompt.earth;systemctl enable rbdmap.service &amp;&amp; systemctl start rbdmap.service
+&prompt.mars;systemctl enable rbdmap.service &amp;&amp; systemctl start rbdmap.service
+</screen>
+      <para>
+       The <filename>/dev/rbd/rbd/sbd01</filename> device should be available
+       on both &sgw;s.
+      </para>
       </step>
       <step>
        <para>
-        Let &si.mars; join the cluster:
+        Initialize the cluster on &si.earth; and let &si.mars; join it. When
```

Comment:
Maybe say "A request to allow &si.mars; join the cluster will appear, when asked..." otherwise it's a bit confusing